docroot/profiles/custom/cgov_site/themes/custom/ncids_trans/front-end/src/legacy/styles/_general.scss

```diff
@@ -77,6 +77,35 @@ h2.related-resources__heading {
   font-style: normal;
 }
 /** */
+// A11Y Updated Video Title Heading
+span.video-title__heading {
+  display: block;
+  font-family: Poppins, Helvetica Neue, Helvetica, Roboto, Arial, sans-serif;
+  font-size: 1.125em;
+	font-weight: normal;
+  margin-top: 1em;
+  margin-bottom: 0.5em;
+	line-height: 24px;
+	outline: none;
+}
```

Comment:
Please make a copy of https://github.com/NCIOCPL/cgov-digital-platform/blob/develop/docroot/profiles/custom/cgov_site/themes/custom/cgov/src/styles/_videos.scss and put it in the ncids_trans legacy styles. This is because this is where the video headings are.

This should use USWDS mixins too to set the styles.